pkg/deb/Makefile

```diff
@@ -46,7 +46,7 @@ horizon-deb: require-version
 # Remember to up VERSION above. If building the deb on mac, first: brew install dpkg
 horizon-cli-deb: require-version
 	opsys=Linux arch=$(arch) $(MAKE) -C ../.. cli/hzn
-	mkdir -p horizon-cli/usr/horizon/{bin,cluster} horizon-cli/etc/bash_completion.d horizon-cli/usr/share/man/man1
+	mkdir -p horizon-cli/usr/horizon/bin horizon-cli/usr/horizon/bin/cluster horizon-cli/etc/bash_completion.d horizon-cli/usr/share/man/man1
```

Comment:
@linggao I know this is merged, but I think you might have a typo here. I think it's supposed to create `horizon-cli/usr/horizon/cluster` not `horizon-cli/usr/horizon/bin/cluster`.